views/login.hbs

```diff
@@ -2,10 +2,10 @@
 
     <div class="container">
         <div class="col-sm-10 col-sm-offset-1">
-            <h1 class="display-4"> {{ header }} </h1>
+            <h1 class="display-4" id="header"> {{ header }} </h1>
             <br>
 
```

Comment:
This doesnt really matter though, but why the break after the br tag? A comment would have been better

Reply:
It's for easier readability.